src/index/index.c

```diff
@@ -152,6 +152,22 @@ RSDoc *Index_IndexGraphEntity
 	return doc;
 }
 
+IndexField IndexField_New
+(
+	char *name,
+	double weight,
+	bool nostem,
+	char *phonetic
+) {
```

Comment:
Another maybe a bit cheaper design could be:
 ```c
void IndexField_New
(
	IndexField *field,
	char *name,
	double weight,
	bool nostem,
	char *phonetic
) {
```
Where the function initialize given `field` input, this can save some mem copies